include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -1070,8 +1098,8 @@ namespace internal
         // get_dof_indices() also for artificial faces to determine if a face
         // is artificial.
         types::global_dof_index invalid_index = numbers::invalid_dof_index;
-        for (; index < dof_indices.size(); ++index)
-          dof_processor(invalid_index, dof_indices[index]);
+        for (; index < get_array_length(dof_indices); ++index)
+          dof_processor(invalid_index, get_array_entry(dof_indices, index));
```

Comment:
Is this really what you want? That if you pass a scalar, that nothing happens?

Reply:
Yes, I want nothing to happen here - this code is there to support cases where we request dof indices on entities that do not carry them, but when I come into this function with the empty type (I prefer that speak over `int` now because `int` is indeed confusing), then I want nothing to happen.